### PR TITLE
Adding without header in final board - Finishes mumuki/mumuki-laboratory#1051

### PR DESCRIPTION
**Pull request by rodri042**
Finishes https://github.com/mumuki/mumuki-laboratory/issues/1051

Depends on https://github.com/mumuki/mumuki-bibliotheca/pull/155 and https://github.com/mumuki/mumuki-gobstones-runner/pull/115

### CODE REVIEW

**Comment by flbulgarelli** [comment]
@rodri042 what will happen if an outdated `gs-board` component - that is, an old version of it - has an attribute that it does not support, like `without-header`? Will it crash or just will it ignore it? 

I want to understand if merging this PR could break backward compatibility with older versions of the runner or not.

File: lib/bibliotheca/exercise.rb [hunk 1]
```diff
@@ -63,22 +63,24 @@ def markdownified
...
-      self.initial_state = to_gs_board(first_example['initial_board'])
-      self.final_state = to_gs_board(first_example['final_board']) || boom_board
+
+      self.initial_state = to_gs_board(first_example['initial_board'], true)
+      self.final_state = to_gs_board(first_example['final_board'], with_head) || boom_board
     end
 
-    def to_gs_board(board)
-      "<gs-board> #{board} </gs-board>" if board
+    def to_gs_board(board, with_head)
```
Comment (flbulgarelli) [inline lib/bibliotheca/exercise.rb]: Just a notice for the future:  _we need to extract this code to another module_.

**Comment by rodri042** [comment]
It'll ignore it in that case (polymer would try to match the attribute with
a nonexisting withoutAttire property, and when it can't found it, will do
nothing)

On Jun 5, 2018 12:17 PM, "Franco Leonardo Bulgarelli" <
notifications@github.com> wrote:

*@flbulgarelli* approved this pull request.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/mumuki/mumuki-bibliotheca-api/pull/114#pullrequestreview-126020678>,
or mute the thread

**Comment by flbulgarelli** [comment]
Awesome, that means we can safely merge this PR.

**Pull request merged**